corehq/apps/users/models.py

```diff
@@ -465,6 +465,30 @@ class DomainMembershipError(Exception):
     pass
 
 
+class DomainPermissionsMirrorSource(models.Model):
+    # Name of the controlling domain
+    name = models.CharField(max_length=126, db_index=True, unique=True)
```

Comment:
Can this be a column in the other table? Especially since it looks like neither table is ever called without joining the two

Reply:
https://github.com/dimagi/commcare-hq/pull/27510/commits/93eb46610c761b27345fb239b935fb564a9f617d